### DISCUSSION TITLE
GLaDOS 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#114] [模板讨论]
GLaDOS 模板问题反馈评论区。
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
2023-07-29 12:30:08
日志：Failed at 1/3 request,
Fail assert: {"re": "oops, token error", "from": "content"} from failed_asserts,
Request URL: https://glados.rocks/api/user/checkin

**Reply by wjf0214** [reply]
更新了
